CHANGELOG.md

```diff
@@ -26,6 +26,28 @@ CHANGELOG
 Swift Next
 ----------
 
+* [SR-8974][]:
+
+  Duplicate tuple element labels are no longer allowed, as they lead to
+  undefined behavior. For example:
```

Comment:
Yikes! Is it really undefined? I would have expected it to always pick the first element in most cases.

Reply:
Yeah, that's what it does in most cases (not sure about its behaviour in past releases though). I should probably not call it "undefined" then - maybe just "because it leads to incorrect behaviour".

Reply:
Sure, that sounds less scary.

Reply:
@jrose-apple Done 👍